src/rolodex/rolodexPrinter.js

```diff
@@ -1,11 +1,11 @@
 import people from './people.json';
 
-people.forEach(function (person) {
-  const names = person.name.split(' ');
-  const firstName = names[0];
-  const lastName = names[1];
-  const email = person.email;
-  const phone = person.phone;
+people.forEach((person) => {
```

Comment:
```suggestion
people.forEach(({ email, phone, name }) => {
```
You can destructure your arguments. Saves a few lines, but isn't always the most readable 🤷‍♀️